### PR TITLE
[int-tests] Add path trailing slash test

### PR DESCRIPTION
**Pull request by suksw**
### Purpose
Integration tests for https://github.com/wso2/product-microgateway/pull/2865
For choreo branch

### Issues

Fixes https://github.com/wso2/product-microgateway/issues/2864

### Automation tests
 - Unit tests added: Yes/No
 - Integration tests added: Yes/No

### Tested environments

Not Tested

---
#### Maintainers: Check before merge
- [x] Assigned 'Type' label
- [ ] Assigned the project
- [x] Validated respective github issues
- [ ] Assigned milestone to the github issue(s)

### CODE REVIEW

File: integration/test-integration/src/test/java/org/wso2/choreo/connect/tests/testcases/standalone/apiDeploy/TrailingSlashTestCase.java [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+
+    @BeforeClass(description = "Get Prod token")
+    void start() throws Exception {
+        jwtTokenProd = TokenUtil.getJwtForPetstore(TestConstant.KEY_TYPE_PRODUCTION, null, false);
+        headers.put(HttpHeaders.AUTHORIZATION, "Bearer " + jwtTokenProd);
+    }
+
+    @Test(description = "test path with trailing slash but without path parameters")
+    public void testTrailingSlashWithoutPathParam() throws CCTestException, MalformedURLException {
+        String endpoint = Utils.getServiceURLHttps(API_CONTEXT + "/pet/findByStatus");
```
Comment (pubudu538) [inline integration/test-integration/src/test/java/org/wso2/choreo/connect/tests/testcases/standalone/apiDeploy/TrailingSlashTestCase.java:51]: Need the same test case here also.

File: integration/test-integration/src/test/java/org/wso2/choreo/connect/tests/testcases/standalone/apiDeploy/TrailingSlashTestCase.java [hunk 2]
```diff
@@ -0,0 +1,93 @@
...
+
+    @Test(description = "test path with trailing slash with multiple path parameters")
+    public void testTrailingSlashWithMultiplePathParams() throws CCTestException, MalformedURLException {
+        String endpoint = Utils.getServiceURLHttps(API_CONTEXT + "/store/1/pet/123");
+        HttpResponse response = HttpsClientRequest.doGet(endpoint, headers);
+        Assert.assertNotNull(response);
+        Assert.assertEquals(response.getResponseCode(), HttpStatus.SC_OK,"Response code mismatched");
+        Assert.assertEquals(response.getData(), ResponseConstants.STORE_INVENTORY_RESPONSE, "Response Body Mismatch.");
+
+        String endpoint_with_slash = Utils.getServiceURLHttps(API_CONTEXT + "/store/1/pet/123");
```
Comment (VirajSalaka) [inline integration/test-integration/src/test/java/org/wso2/choreo/connect/tests/testcases/standalone/apiDeploy/TrailingSlashTestCase.java:87]: Here, it should end with a / ?

Comment (suksw) [reply integration/test-integration/src/test/java/org/wso2/choreo/connect/tests/testcases/standalone/apiDeploy/TrailingSlashTestCase.java]: Fixed with https://github.com/wso2/product-microgateway/pull/2877/commits/b5b41e57906ff728fcda6d305ef7195b7e992146

File: integration/test-integration/src/test/java/org/wso2/choreo/connect/tests/testcases/standalone/apiDeploy/TrailingSlashTestCase.java (hunk 1 above)
Comment (suksw) [reply integration/test-integration/src/test/java/org/wso2/choreo/connect/tests/testcases/standalone/apiDeploy/TrailingSlashTestCase.java]: Fixed with https://github.com/wso2/product-microgateway/pull/2877/commits/b5b41e57906ff728fcda6d305ef7195b7e992146 and https://github.com/wso2/product-microgateway/pull/2877/commits/10431d907d6974a745483c45c5b7a89aad3eec10

**Review by pubudu538 — APPROVED** [review]

**Comment by choreo-cicd** [comment]
[succeeded] Controlplane cluster : dev-deployment-v2 : 20220603.5

**Comment by choreo-cicd** [comment]
[succeeded] DataplaneE1 cluster : dev-deployment-v2 : 20220603.5

**Comment by choreo-cicd** [comment]
[succeeded] DataplaneE2 cluster : dev-deployment-v2 : 20220603.5

**Comment by choreo-cicd** [comment]
[succeeded] DataplaneE1 cluster : prod-deployment-v2 : 20220608.1

**Comment by choreo-cicd** [comment]
[succeeded] Controlplane cluster : prod-deployment-v2 : 20220608.1

**Pull request merged**